samples/Samples.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -37,8 +36,6 @@
 
         "OTEL_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
         "OTEL_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
-        "OTEL_VERSION": "1.0.0",
-        "OTEL_SERVICE": "Samples.AspNetCoreMvc31",
```

Comment:
Did you intend to just change `OTEL_SERVICE` to `OTEL_SERVICE_NAME`?

Reply:
Addressed fc34fdfb